descope/auth/sdk.go

```diff
@@ -42,18 +46,11 @@ type MagicLink interface {
 	SignUpOrInCrossDevice(method DeliveryMethod, identifier string, URI string) (*MagicLinkResponse, error)
 
 	// GetSession - Use to get a session that was generated by SignIn/SignUp request, and verified with Verify request.
-	// This is a shortcut for GetSessionWithOptions(method, code, WithResponseOption(w))
 	GetSession(pendingRef string, w http.ResponseWriter) (*AuthenticationInfo, error)
-	// GetSessionWithOptions - Use to get a session that was generated by SignInMagicLink/SignUpMagicLink request, and verified with VerifyMagicLink request.
```

Comment:
Need to add this comment back

Reply:
Why? I have the same exact comment above